vars/setupCommonPipelineEnvironment.groovy

```diff
@@ -38,9 +39,10 @@ import groovy.transform.Field
      * `customDefaults`, but from local or remote files instead of library resources. They are merged with and
      * take precedence over `customDefaults`.*/
     'customDefaultsFromFiles',
-    /** The map returned from a Jenkins git checkout. Used to set the git information in the
-     * common pipeline environment */
-    'scmInfo'
+    /**
+      * The projects git repo url. Typically the fetch url.
```

Comment:
This url could include either https or ssh url right?

Reply:
To my understanding: yes. The URLs we need internally are calculated [here](https://github.com/SAP/jenkins-library/blob/master/vars/setupCommonPipelineEnvironment.groovy#L224) based on the protocol. Nota bene: the concept of having a gitHttpsUrl and a gitSshUrl is weird since git knows a fetch and a push url, but not per se a protocol specific url. But that is another story.